pkgs/applications/office/paperless-ngx/default.nix

```diff
@@ -240,7 +251,8 @@ python.pkgs.buildPythonApplication rec {
   ]
   ++ redis.optional-dependencies.hiredis
   ++ twisted.optional-dependencies.tls
-  ++ uvicorn.optional-dependencies.standard;
+  ++ uvicorn.optional-dependencies.standard
+  ++ lib.optional stdenv.hostPlatform.isx86 zxing_cpp; # Upstream only has this for x86 systems
```

Comment:
Builds fine on aarch64-linux. What is the reason upstream has this only for x86?

Reply:
[The docs](https://docs.paperless-ngx.com/configuration/) only state 

> `PAPERLESS_CONSUMER_BARCODE_SCANNER=<string>`
> [...]
> zxing is not available on all platforms

But when you have tested it (I sadly don't have any arm aarch hosts yet, so I wasn't able to test it) we can also include it for every system.

Reply:
Let's include it unconditionally then.

Reply:
For reference: This is why `zxing` isn't included for all architectures: https://github.com/paperless-ngx/paperless-ngx/pull/2903#issuecomment-1472714824